docs/src/main/tut/semigroup.md

```diff
@@ -83,13 +83,38 @@ two |+| n
 ```
 
 You'll notice that instead of declaring `one` as `Some(1)`, I chose
-`Option(1)`, and I added an explicit type declaration for `n`. This is
-because there aren't type class instances for Some or None, but for
-Option. If we try to use Some and None, we'll get errors:
+`Option(1)`, and I added an explicit type declaration for `n`. This is because
+the type class instance for `Semigroup` is defined for `Option`. There is no `Semigroup`
+instance defined for `None` or `Some`--you will get errors if you attempt to combine a
+`Some` or `None` instance:
+
+```tut:scala
```

Comment:
This should be `tut:book`. Changing this and the same below, should make the build pass.